### PR TITLE
Update get_started tutorial about deploying on ascend platform

### PR DESCRIPTION
**Pull request by jinminxi104**
added graph mode and w4a16.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started/ascend/get_started.md [hunk 1]
```diff
@@ -18,13 +18,15 @@ cd lmdeploy
 
 The Docker version is supposed to be no less than `18.03`. And `Ascend Docker Runtime` should be installed by following [the official guide](https://www.hiascend.com/document/detail/zh/mindx-dl/60rc2/clusterscheduling/clusterschedulingig/.clusterschedulingig/dlug_installation_012.html).
 
+***If error message `libascend_hal.so: cannot open shared object file` shows, that means **Ascend Docker Runtime** is not installed correctly!***
```
Comment (grimoire) [inline docs/en/get_started/ascend/get_started.md:21]: block quote is a good format to display these messages.

> [!WARNING]  
> https://github.com/orgs/community/discussions/16925

**Review by grimoire — APPROVED** [review]

**Review by CyCle1024 — APPROVED** [review]

File: docs/en/get_started/ascend/get_started.md [hunk 2]
```diff
@@ -69,41 +75,57 @@ from lmdeploy import pipeline, PytorchEngineConfig
 from lmdeploy.vl import load_image
 if __name__ == "__main__":
     pipe = pipeline('OpenGVLab/InternVL2-2B',
-                    backend_config=PytorchEngineConfig(tp=1, device_type='ascend'))
+                    backend_config=PytorchEngineConfig(tp=1, device_type='ascend', eager_mode = True))
```
Comment (lvhan028) [inline docs/en/get_started/ascend/get_started.md:78]: Please remove the spaces on both sides of the equals sign.

File: docs/en/get_started/ascend/get_started.md [hunk 3]
```diff
@@ -54,7 +60,7 @@ from lmdeploy import pipeline
 from lmdeploy import PytorchEngineConfig
 if __name__ == "__main__":
     pipe = pipeline("internlm/internlm2_5-7b-chat",
-                    backend_config = PytorchEngineConfig(tp=1, device_type="ascend"))
+                    backend_config = PytorchEngineConfig(tp=1, device_type="ascend", eager_mode = True))
```
Comment (lvhan028) [inline docs/en/get_started/ascend/get_started.md:63]: Please remove the spaces on both sides of the equals sign.

**Comment by lvhan028** [comment]
linting faied. Please run the following commands to resolve it.
```
pip install pre-commit
cd lmdeploy # the root path of lmdeploy repo
pre-commit install
pre-commit run --all-files
```

File: docs/zh_cn/get_started/ascend/get_started.md [hunk 4]
```diff
@@ -17,13 +17,15 @@ cd lmdeploy
 
 Docker 版本应不低于 18.03。并且需按照[官方指南](https://www.hiascend.com/document/detail/zh/mindx-dl/60rc2/clusterscheduling/clusterschedulingig/clusterschedulingig/dlug_installation_012.html)安装 Ascend Docker Runtime。
 
+***如果在后续容器内出现`libascend_hal.so: cannot open shared object file`错误，说明Ascend Docker Runtime没有被正确安装***
```
Comment (lvhan028) [inline docs/zh_cn/get_started/ascend/get_started.md:20]: block quote is recommended

File: docs/zh_cn/get_started/ascend/get_started.md [hunk 5]
```diff
@@ -45,6 +47,8 @@ docker run -e ASCEND_VISIBLE_DEVICES=0 --rm --name lmdeploy -t lmdeploy-aarch64-
 
 ## 离线批处理
 
+***图模式已经支持了Atlas 800T A2。目前，单卡下的InternLM2-7B/LLaMa2-7B/Qwen2-7B已经通过测试。用户可以设定`eager_mode = False`来开启图模式，或者设定`eager_mode = True`来关闭图模式。(启动图模式需要事先source `/usr/local/Ascend/nnal/atb/set_env.sh`)***
```
Comment (lvhan028) [inline docs/zh_cn/get_started/ascend/get_started.md:50]: block quote is recommended

File: docs/zh_cn/get_started/ascend/get_started.md [hunk 6]
```diff
@@ -69,41 +73,56 @@ from lmdeploy import pipeline, PytorchEngineConfig
...
-                    backend_config=PytorchEngineConfig(tp=1, device_type='ascend'))
+                    backend_config=PytorchEngineConfig(tp=1, device_type='ascend', eager_mode=True))
     image = load_image('https://raw.githubusercontent.com/open-mmlab/mmdeploy/main/tests/data/tiger.jpeg')
     response = pipe(('describe this image', image))
     print(response)
 ```
 
 ## 在线服务
 
+***图模式已经支持Atlas 800T A2。目前，单卡下的InternLM2-7B/LLaMa2-7B/Qwen2-7B已经通过测试。
```
Comment (lvhan028) [inline docs/zh_cn/get_started/ascend/get_started.md:84]: block quote is recommended

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**